ios/FluentUI/Navigation/NavigationBar.swift

```diff
@@ -53,7 +53,11 @@ open class NavigationBarTopSearchBarAttributes: NavigationBarTopAccessoryViewAtt
 /// Contains the MSNavigationTitleView class and handles passing animatable progress through
 /// Custom UI can be hidden if desired
 @objc(MSFNavigationBar)
-open class NavigationBar: UINavigationBar {
+open class NavigationBar: UINavigationBar, TwoLineTitleViewDelegate {
+    public func twoLineTitleViewDidTapOnTitle(_ twoLineTitleView: TwoLineTitleView) {
```

Comment:
we should have comments for all public functions for documentation

Reply:
Arguably, said documentation belongs on `TwoLineTitleViewDelegate` itself. That can be added.

Reply:
is this overriable for the clients who override navigationbar? still have comments here please

Reply:
Comment added. For what it's worth, clients who override `NavigationBar` can't override this because this method is public, not open.